CHANGELOG.rst

```diff
@@ -26,6 +26,6 @@ Dependencies version where updated to:
 - **kinto 2.0.0**: https://github.com/Kinto/kinto/releases/tag/2.0.0
 - **kinto-attachment 0.4.0**: https://github.com/Kinto/kinto-attachment/releases/tag/0.4.0
 - **kinto-changes 0.1.0**: https://github.com/Kinto/kinto-changes/releases/tag/0.1.0
-- **kinto-signer 0.1.0**: https://github.com/Kinto/kinto-signer/releases/tag/0.1.0
+- **kinto-signer 0.2.0**: https://github.com/Kinto/kinto-signer/releases/tag/0.2.0
```

Comment:
We should not change that in the changelog, when we released version 0.1.0 it was kinto-signer 0.1.0